### DISCUSSION TITLE
Rich text editor

### DISCUSSION THREAD

**Discussion by ghost** [zauberzeug/nicegui#1179] [Q&A] [answered]
### Question

Hi,

Is there a way to include rich text editor like quill in the web page to edit the content.

Thank you

**Comment by rodja** [comment, marked as answer]
We would love to see such element in NiceGUI. Very similar to the request for the WYSIWYG editor component (#1124). Does anyone want to give it a try? A pull request would be very welcome.

**Reply by rodja** [reply]
On second thought, I think all WYSIWYG editors can be considered rich text editors because they support complex formatting, but not all rich text editors are WYSIWYG editors, because they might not provide real-time preview of the final output. Therefore I'll close this Q&A in favour of the feature request #1124.
